### ISSUE TITLE
Implement Free NTLM authentication module

### ISSUE THREAD

**Issue by i00** [jc21/nginx-proxy-manager#1138]
Only Ngnix Plus seems to support NTLM (even then there would be no support in the web interface for Nginx Proxy Manager)
https://github.com/gabihodoroaga/nginx-ntlm-module

It would be nice if free NTLM support could be added such as this module:
https://github.com/gabihodoroaga/nginx-ntlm-module

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

The ability to add NTLM authentication support

**Describe alternatives you've considered**

N/A

**Additional context**

**Comment by chaptergy** [comment]
I find it highly unlikely this will ever be implemented, since this seems to be a rather niche SSO, but it would broadly fit under the umbrella of https://github.com/jc21/nginx-proxy-manager/issues/437

**Comment by i00** [comment]
A lot of Windows on-premise things, such as TFS, Windows Admin Center etc

**Comment by danfratamico** [comment]
> A lot of Windows on-premise things, such as TFS, Windows Admin Center etc

I completely agree, I would benefit so much from NTLM support being added in NPM. Exchange web services are also broken because of this.

**Comment by CmdLnInt** [comment]
Is it possible to use nginx/openresty reverse proxy ntlm support?

**Comment by KevinCA-44** [comment]
Has there been any news about adding an NTLM module to NPM since ?
We have many services who work with NTLM and we don't have found any options for fix it.

**Comment by danfratamico** [comment]
Following up on this, any update?

**Comment by GodfatherX64** [comment]
Following up on this, any update?

**Comment by GodfatherX64** [comment]
A needed feature, appreciated if you consider it

**Comment by TrentSteenholdt** [comment]
https://blog.cortanadesign.com.au/2025/09/10/how_to_get_exchange_2019_working_behind_nginx_proxy_manager
